packages/charts/src/core/plugins/gauge/donut-gauge-labels-plugin.ts

```diff
@@ -82,15 +82,10 @@ export class DonutGaugeLabelsPlugin extends ChartPlugin {
     private drawThresholdLabels() {
         const thresholdsSeries = this.chart.getDataManager().chartSeriesSet.find((series: IChartSeries<IAccessors<any>>) =>
             series.renderer instanceof DonutGaugeThresholdsRenderer);
-        const renderer = (thresholdsSeries?.renderer as DonutGaugeThresholdsRenderer);
```

Comment:
I basically rearranged things here and added a warning with a short circuit if the threshold series is undefined. If you try to add the labels plugin without defining a series for the thresholds, you'll get a warning instead of a crash.